language/German/strings.po

```diff
@@ -2112,7 +2156,7 @@ msgstr "Software-Medien"
 #: src/frontend/mame/ui/dirmenu.cpp:46
 msgctxt "path-option"
 msgid "Sound Samples"
-msgstr ""
+msgstr "Sound-Samples"
```

Comment:
Since we're translating everything, shouldn't that be something like "Sounddateien"?

Reply:
There are other places refering to the _samples_ as _samples_ already, so unchanged for consistency.